C-Plus-Plus/cp/Dividing_task.cpp

```diff
@@ -0,0 +1,44 @@
+// This is Competitve Programming  question
+
+// Dividing Tasks to student is CP problem that state that There are N students in the class, i'th student has Ci task in his collection. Your task is to determine if it is possible to redistribute C1 + C2 + ... + Cn task among the student of class thus that i'th student would get i task.
+
+// given number of students (n)
+// given the task assigned to them
+// output if the problem state is possible if true print Yes else No
+
+// Algorithm
+// declare and initialise sum_of_task that means sum of task given in array to 0
+// declare and initialise sum_of_i_student means sum of ith student to 0
+// if sum_of_task is equal to sum_of_i_student is equal print YES else NO
+#include <bits/stdc++.h>
+using namespace std;
+int main()
```

Comment:
Don't put the code logic in `main()` function. Keep it in another function and use `main()` for the driver code